parachain/node/src/rpc.rs

```diff
@@ -96,13 +97,13 @@ where
 }
 
 /// Full client dependencies
-pub struct FullDeps<C, P, A: ChainApi> {
+pub struct FullDeps<C, P> {
 	/// The client instance to use.
 	pub client: Arc<C>,
 	/// Transaction pool instance.
 	pub pool: Arc<P>,
 	/// Graph pool instance.
-	pub graph: Arc<Pool<A>>,
+	pub graph: Arc<P>,
```

Comment:
even checked the Astar upper stream, still don't understand why both `pool` and `graph` exists together. Normally only one would be enouth.